src/Symfony/Component/Serializer/Tests/Normalizer/DataUriNormalizerTest.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Tests\Normalizer;
+
+use Symfony\Component\HttpFoundation\File\File;
+use Symfony\Component\Serializer\Normalizer\DataUriNormalizer;
+
+/**
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class DataUriNormalizerTest extends \PHPUnit_Framework_TestCase
+{
+    const TEST_GIF_DATA = 'data:image/gif;base64,R0lGODdhAQABAIAAAP///////ywAAAAAAQABAAACAkQBADs=';
+    const TEST_TXT_DATA = 'data:text/plain,K%C3%A9vin%20Dunglas%0A';
+    const TEST_TXT_CONTENT = "Kévin Dunglas\n";
+
+    /**
+     * @var DataUriNormalizer
+     */
+    private $normalizer;
+
+    public function setUp()
+    {
+        $this->normalizer = new DataUriNormalizer();
+    }
+
+    public function testInterface()
+    {
+        $this->assertInstanceOf('Symfony\Component\Serializer\Normalizer\NormalizerInterface', $this->normalizer);
+        $this->assertInstanceOf('Symfony\Component\Serializer\Normalizer\DenormalizerInterface', $this->normalizer);
```

Comment:
I agree with you that it's not very useful but it guaranties that nobody remove the `implements` or an interface.